english/open-door/open-door.md

```diff
@@ -6,3 +6,7 @@ outside the room you see...
 
 That you are about to be crushed by a giant corn. :)
 
+     or 
+
+[The nose wheel light of a 747 that is landing directly in front of you](../airplane-princess/slide.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.